src/CTA.FeatureDetection.AuthType/CompiledFeatures/WebConfigFeature.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Codelyzer.Analysis;
+using CTA.FeatureDetection.Common.Models.Features.Base;
+using CTA.Rules.Config;
+
+namespace CTA.FeatureDetection.AuthType.CompiledFeatures
+{
+    public abstract class WebConfigFeature : CompiledFeature
+    {
+        private const string WebConfigFileName = Rules.Config.Constants.WebConfig;
```

Comment:
Do we really need a local variable here for this constant? Since it is not being changed can we just reference it directly from the Constants class?

Reply:
Currently doing some refactoring- this will be removed